scripts/library.sh

```diff
@@ -543,6 +544,12 @@ function go_update_deps() {
 
   if (( UPGRADE )); then
     echo "--- Upgrading to ${VERSION}"
+    if [ -z ${GOPROXY+x} ]; then
```

Comment:
What is `+x`?

Reply:
${var+x} is a parameter expansion which evaluates to nothing if var is unset, and substitutes the string x otherwise.

Reply:
This could absolutely use a comment 🤣

Reply:
Added.